### ISSUE TITLE
assert database could not be found?

### ISSUE THREAD

**Issue by zhengying** [Chris85bh/Navigation2D#1]
I think your image meta data file (.meta) not upload
so sprite missing everywhere.
the unity report warning : "Rebuilding Library because the asset database could not be found!"
My unity version : unity2018.03

**Comment by briermay** [comment]
yes you need to set unity to non hidden meta data or else this is pretty much useless as you have nothing showing how to use this
